src/main/java/com/jensjansson/ce/views/main/MainView.java

```diff
@@ -63,74 +62,63 @@ private Component createSideMenu() {
         H2 header = new H2("Collaboration Engine Demo");
         header.addClassNames("flex", "items-center", "h-xl", "m-0", "px-m",
                 "text-m");
+
         menu = createMenuLinks();
-        HorizontalLayout avatar = createMenuAvatar();
+        Footer footer = createMenuFooter();
 
-        Div layout = new Div(header, menu, avatar);
-        layout.addClassNames("bg-contrast-5", "flex", "flex-col",
-                "items-strech", "min-h-full");
-        return layout;
+        com.vaadin.flow.component.html.Section section = new com.vaadin.flow.component.html.Section(header, menu, footer);
```

Comment:
Use import statement instead of inline defining Section import

Reply:
Never mind. Class extends AppLayout, and thus AppLayout.Section takes always precedence in the imports.